typedoc.client.json

```diff
@@ -1,14 +1,14 @@
 {
-  "exclude": ["**/node_modules/**", "**/protocols/*.ts", "**/endpoints.ts", "**/ruleset.ts"],
```

Comment:
`**/ruleset.ts` needs to still be included

Reply:
Addressed in ce91847e4567abe5aaa01707cc7a347694f12a18